awx/main/models/projects.py

```diff
@@ -431,13 +489,16 @@ def notification_templates(self):
         if self.organization is not None:
             error_notification_templates = set(error_notification_templates +
                                                list(base_notification_templates
-                                                    .filter(organization_notification_templates_for_errors=self.organization)))
+                                                    .filter(
```

Comment:
There's a bunch of unrelated formatting changes in this PR. Please remove them.

Reply:
Thanks, for some reason those changes were recommended when tested on a local build, I just undid them and the Zuul tests passed.